### PR TITLE
SK-1234: Load test

### PR DESCRIPTION
**Pull request by usmangeek**
Current working version for testing email attestations flow. Locally, I have used [k6](https://k6.io/docs/) from Grafana Labs to simulate email attestations by 20 users in 20 seconds.

### CODE REVIEW

File: src/backend/loadTest/loadTest.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  const email = generateRandomEmail();
+
+  await quoteEmailApi({ email }, sessionId);
+
+  const identityDid =
+    'did:kilt:4qsQ5sRVhbti5k9QU1Z1Wg932MwFboCmAdbSyR6GpavMkrr3';
+
+  const claim = Claim.fromCTypeAndClaimContents(
+    emailCType,
+    { Email: email.trim() },
```
[the remark this answers is not in the document]
Comment (usmangeek) [reply src/backend/loadTest/loadTest.ts:80]: Hahaha! O yes!

File: scripts/loadTest/test.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
+/* eslint-disable import/no-unresolved */
```
Comment (arty-name) [inline scripts/loadTest/test.ts:1]: I think we should update the configuration for this rule, such one-off overrides for eslint or tsc are not a good sign.

**Review by arty-name — COMMENTED** [review]
This PR is missing the deactivation of actual email sending during the test. Without it we will try to send out hundreds of undeliverable emails and risk being blocked by AWS.

**Comment by usmangeek** [comment]
> This PR is missing the deactivation of actual email sending during the test. Without it we will try to send out hundreds of undeliverable emails and risk being blocked by AWS.

Sure, we will do this

File: scripts/loadTest/test.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+  const claim = Claim.fromCTypeAndClaimContents(
+    emailCType,
+    { Email: email },
+    'did:kilt:4qsQ5sRVhbti5k9QU1Z1Wg932MwFboCmAdbSyR6GpavMkrr3',
+  );
+
+  const dAppDid = dAppEncryptionKeyUri.split('#')[0];
+
+  const encryptedMessage = await getEncryptedMessage(claim, dAppDid);
```
[the remark this answers is not in the document]
Comment (usmangeek) [reply scripts/loadTest/test.ts:105]: We want to get encrypted signed request for attestation from Sporran, too.

**Review by arty-name — APPROVED** [review]

**Pull request merged**